app/presenters/tree_node_builder_compliance_history.rb

```diff
@@ -1,11 +1,15 @@
 class TreeNodeBuilderComplianceHistory < TreeNodeBuilder
+  def expand_node?
+    options[:open_all] && options[:expand] != false
+  end
+
   def hash_node
     text = object[:text].html_safe
     @node = {:key   => build_hash_id,
              :icon  => node_icon("#{object[:image]}.png"),
              :title => text}
     # Start with all nodes open unless expand is explicitly set to false
-    @node[:expand] = true if options[:open_all] && options[:expand] != false
+    @node[:expand] = expand_node?
```

Comment:
Not `true` or `false` is assigned, previously `true` was assigned or the value was unchanged.

Is this intended and tested?